protobuf/can_do_crypto.proto

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2021 Contributors to the Parsec project.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+syntax = "proto3";
+
+package can_do_crypto;
+
+import "psa_key_attributes.proto";
+
+enum CheckType {
+  Use = 0;
```

Comment:
I think we usually leave aside the variant with value 0 to be some "default" value (in other enums we seem to have called them `<Enum name>_NONE`). This avoids the possibility of any value being selected non-explicitly, just by virtue of the field missing in the message - clients must send a specific code, explicitly